### DISCUSSION TITLE
aerodynamic free area

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13912] [Q&A]
![image](https://github.com/user-attachments/assets/aa52c38b-c6d9-437f-925d-25c1bcc768ac)

If I know the free area of an openable vent at the head of stair is 1m2. The aerodynamic free area is 0.7m2 as the coefficient of discharge is 0.7. How can I model this realistically for pressure drop. Should I model a 1m2 hole opening, or is there a way to specify coefficient of discharge.

**Comment by drjfloyd** [comment]
The aerodynamic area results from the pressure drop due to flow contracting to enter the vent and expanding after it leaves the vent. If you have sufficient grid resolution you will resolve that flow and the pressure drop will happen naturally.  If you are coarsely resolving the opening then you can model it with HVAC where you can specify a duct loss that gives you the equivalent of C_d=0.7

**Comment by ShehabGamrah** [comment]
Thanks. I suppose I have to specify two hvac vents and connect them with a damper type duct. What should I specify for the length and area of duct, and the forward, reverse loss, and roughness type and value.

**Reply by drjfloyd** [reply]
Length should represent the length of the flow path.

The areas should represent the actual duct area.

Set the forward and reverse loss to give you the equivalent of a C_d=0.7.  See the duct equation (11.3) in the Technical Reference Guide. Think about what terms are zero or effectively zero for an ideal orifice and then rearrange the equation to be the same form as the orifce equation. That will give you the equiivalence between K and C_d.

For friction factor. You have a ~1 m diameter and 1 cm wide obstructions in the flow path which looks to be maybe 10 - 20 cm long. This is an e/D a=0.01 giving a friction factor of 0.04.  K due to wall friction is f L/D or a K of 0.004 to 0.008.  Compare this to the K you get for the orifice effect. Does it matter?

**Reply by ShehabGamrah** [reply]
Thanks. I was able to derive relation between loss and coefficient of discharge. For this example should I specify the value for forward, and reverse or one only.

**Reply by drjfloyd** [reply]
LOSS for a duct takes two values. If you do not specify two values, then you have only specified the forward loss. If you do not specify a reverse loss and there is at any point in time during your simulation reverse flow, then it will have no loss applied and you will not get the correct velocity for the pressure difference.
